Updated deprecated features

Abhash-Chakraborty opened the pull request:
Closes: #31

manthanawgan commented:
@Abhash-Chakraborty the changes you did in animatedgroup.tsx file are unnecessary.
<img width="848" height="904" alt="image" src="https://github.com/user-attachments/assets/66de7288-763f-46f0-94b3-ce1d536d38d9" />
<img width="1425" height="836" alt="image" src="https://github.com/user-attachments/assets/979e6c42-cd88-4e1a-a00f-6b3f66b5d069" />

you just deleted the single quotes and added double quotess???

manthanawgan commented:
also I noticed the PR went straight to main without review. For consistency and safety, can we make sure all PRs have at least one reviewer approve before merging?

Abhash-Chakraborty commented:
@manthanawgan

My linter extension auto-formatted the quotes to maintain code consistency, which is standard for the project. The other changes in the PR were:

- Updated the `motion` component instantiation logic in `AnimatedGroup` to use `motion.create` instead of direct function calls, improving flexibility and future compatibility.

- Changed Clerk redirect environment variables to use fallback URLs (`CLERK_SIGN_IN_FALLBACK_REDIRECT_URL` and `CLERK_SIGN_UP_FALLBACK_REDIRECT_URL`) for more robust authentication routing.

- Added the `priority` attribute to key `Image` components in `hero-section.tsx` to ensure critical images are loaded promptly for better user experience. [[1]](diffhunk://#diff-54ae2e5536f783107392a1fb5e26b08402e2346910d30eff030c5550cc51a167R90) [[2]](diffhunk://#diff-54ae2e5536f783107392a1fb5e26b08402e2346910d30eff030c5550cc51a167R197-R205)

Regarding the merge, it was a low-risk optimization, not a feature change. After Copilot's review confirmed there were no issues, I merged it directly to maintain momentum. I performed the final review for this PR myself.
